pkg/controller/storagecluster/external_resources.go

```diff
@@ -205,16 +282,20 @@ func (r *ReconcileStorageCluster) createExternalStorageClusterConfigMap(cm *core
 			err = r.client.Create(context.TODO(), cm)
 			if err != nil {
 				reqLogger.Error(err, "creation of configmap failed")
-				return err
 			}
 		} else {
 			reqLogger.Error(err, "unable the get the configmap")
-			return err
 		}
+		return err
```

Comment:
Don't do this, leave the return statements as-is.

Reply:
Yup, changed it ...